content/building-blocks/automate-and-execute-your-work/automate-your-workflow/ghactions-workflow.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Build a workflow on a GitHub hosted runner" 
```

Comment:
I think that the main takeaway of the BB for users is the implementation of the Testing/Formatting workflow rather than whether it runs on X or Y type of runner and the title would benefit from focusing on this. It could look something like:

"Build GitHub Actions workflow"
"Build a GitHub Actions workflow for code formatting and testing"
"Implement a GitHub Actions workflow for code formatting and testing"

Where "GitHub Actions" is given a central role in the title against references to the kind of runner.